README.md

```diff
@@ -8,6 +8,7 @@ Each of the samples builds from a common base project, in order to sucessfully b
 ### Console Applications
 - [Base console application setup](./base-console-app)
 - [Day 16 - Create a new user](./day16-create-user)
+- [Day NN - Template](./dayNN-template)
```

Comment:
Can you please update this label in the TOC to use the correct day number and scenario

Reply:
We had an email thread going with @BrianTJackett I still don't know what my day number is. Once I get the information I'll update it.

Reply:
@baywet In the current schedule Planner is on Day 22, but don't worry too much about the exact day number.  We have been shifting days around based on readiness of content.  If yours is ready to go sooner we may move it up in the schedule.  If you want to stick with a generic "Day NN - Planner" for time being I'm fine with that.